### ISSUE TITLE
User Friendly Usage Guide

### ISSUE THREAD

**Issue by sahilbabbar** [osxfuse/osxfuse#234]
Hi,

There is no user-friendly usage guide for getting started with osxfuse. In order to access a NTFS external hard-disk, the user has to go through the wiki and find hints and stumble upon some unknown terms to figure out how this thing actually works.

Please link to one if present.

Thanks.

**Comment by LauraLaureus** [comment]
The texte under the title "Free Third-Party Drivers – It’s Free, But Takes Some Extra Work" of the following link may help you: 
http://www.howtogeek.com/236055/how-to-write-to-ntfs-drives-on-a-mac/

**Comment by chiuchuihui** [comment]
I helped several friends setup the NTFS-3G on Mac OS X Snow Leopard, Yosemite, and El Capitan.  I have composed a step-by-step guide for regular users.  How do I contribute?

**Comment by bfleischer** [comment]
@chiuchuihui Thanks, but the wiki already contains a step-by-step guide for NTFS-3G. See  https://github.com/osxfuse/osxfuse/wiki/NTFS-3G. Maybe you could write instructions for a different file system?

**Comment by jamie-arcc** [comment]
not sure that I can contribute much to this, but bumping because I personally could really use this.  I'm coming from linux so a lot of this stuff is alien to me.

**Comment by ipatch** [comment]
@jamie-arcc 

could you be more specific with what you are not understanding?  I don't mind updating the wiki or adding new items to make things more clear if that could help.

**Comment by fuzzy76** [comment]
I think people are looking for something like "getting started with osxfuse". It is not necessarily clear for everyone that you have to go into the filesystem-specific wiki article. At least I was looking for something a bit more introductory.

**Comment by bfleischer** [comment]
@fuzzy76 Feel free to write such a guide. The wiki is writable.

**Comment by NHuebnerWMG** [comment]
Yeh so I installed and reinstalled and it says I'm updating it. I have no idea how to use OSXFuse. It's not in my applications... I tried doing something like "Connect to a server" then typed my ssh:// server, and it took me to a Terminal tab. I've used this in Windows I think a while back but the Mac version kind of dumps you off the side of a cliff.

Feel free to throw your "first time user" approach at me, I will be happy to QA for you.
